### ISSUE TITLE
Facebook Messenger integration

### ISSUE THREAD

**Issue by KevinJCochran** [KevinJCochran/kooking-with-kevin#9]
Add Facebook Messenger integration to the list.

- When an item is added, deleted, or voted on, send a message to the group chat.
- Send a message to the bot to add it to the list.

The second one could be difficult :(

**Comment by matthew-ferreira** [comment]
[https://developers.facebook.com/docs/messenger-platform/getting-started/sample-apps](url)

These might be a good place to start, at least for the latter
Also the "persistent menu" option

It doesn't look like messenger supports bots sending messages to group chats, but it DOES support bots sending 1:1 messages not initiated by the user, as long as they're tagged properly (see [https://developers.facebook.com/docs/messenger-platform/send-messages#message_tags](url))

TLDR: We can use account linking to link our messenger accounts to our KwK accounts, and then the server can send us individual messages for updates and we can request that items be added/deleted from the list, or even vote on items directly from messenger
